### PR TITLE
Fixed a few typos

### PR DESCRIPTION
**Pull request by michaelforrest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Looks for a previously verified token. If one is found, continues, otherwise it
 will call the `:unauthenticated` function of your handler.
 
 When you ensure a session, you must declare an error handler. This can be done
-as part of a pipeline or inside a pheonix controller.
+as part of a pipeline or inside a phoenix controller.
```
Comment (doomspork) [inline README.md]: We should probably capitalize Phoenix.

Comment (michaelforrest) [inline README.md]: True.

Comment (doomspork) [inline README.md]: Thanks :+1:

Comment (doomspork) [inline README.md]: ping @michaelforrest — If you could update "Phoenix" this should be good to merge. :grinning:

**Comment by michaelforrest** [comment]
Ok sorry for the lapse - here it is!

**Comment by doomspork** [comment]
Thanks @michaelforrest, could you please squash? :+1:

**Comment by michaelforrest** [comment]
I'm editing this up on Github, not on a local copy - I don't think I can do a squash from here.

**Comment by hassox** [comment]
so many typos :( I'm the king of them I'm afraid. Thanks so much for going over this.

**Pull request merged**